index.html

```diff
@@ -14,7 +14,7 @@
   </head>
   <body onload="InitMap()">
     <div id="map"></div>
-    <div>
+    <div id="address">
       <input type="textbox" name="Address" id="address">
```

Comment:
I see why you wanted this - so the style puts it in the same place as the map. Just use another string and it'll be fine :)